docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -825,7 +978,7 @@
 
       <para>The default envrionment.conf:</para>
 
-      <para><programlisting>## HPCC Systems default environment configuration file 
+      <para><programlisting>## HPCC Systems<superscript>®</superscript> default environment configuration file 
```

Comment:
You cannot alter the display of the conf file's test here unless you have also edited the default environment.conf file